reseq_ros2/reseq_ros2/agevar.py

```diff
@@ -98,24 +106,24 @@ def yaw_feedback_callback(self, msg, module_num):
 
     # given data of a module, compute linear and angular velocities of the next one
     def kinematic(self, linear_vel, angular_vel, yaw_angle):
-        linear_out = linear_vel * cos(yaw_angle) + rc.a * angular_vel * sin(yaw_angle)
-        angular_out = (linear_vel * sin(yaw_angle) - rc.a * angular_vel * cos(yaw_angle)) / rc.b
+        linear_out = linear_vel * cos(yaw_angle) + self.a * angular_vel * sin(yaw_angle)
+        angular_out = (linear_vel * sin(yaw_angle) - self.a * angular_vel * cos(yaw_angle)) / self.b
         
         return linear_out, angular_out
 
     # compute motors velocity for each module
     def vel_motors(self, lin_vel, ang_vel, sign):
-        wdx = (lin_vel+ang_vel*rc.d/2) / rc.r_eq
-        wsx = (lin_vel-ang_vel*rc.d/2) / rc.r_eq
+        w_right = (lin_vel+ang_vel*rc.d/2) / rc.r_eq
+        w_left = (lin_vel-ang_vel*rc.d/2) / rc.r_eq
```

Comment:
```suggestion
        w_right = (lin_vel+ang_vel*self.d/2) / self.r_eq
        w_left = (lin_vel-ang_vel*self.d/2) / self.r_eq
```

Reply:
For the next time, to avoid these mistakes it is better to deal with conflicts during the merge phase, and not in the middle of the PR 😄